fe/index.php

```diff
@@ -0,0 +1,44 @@
+<?php
+ require_once '../be/dbcon.php';
+ require_once '../be/functions.php';
+
+ $db = connectTodb();
+
+ //var_dump($db);
+
+ //var_dump(getAllRecords($db));
+$records = getAllRecords($db);
+$recordHtml = displayRecords($records);
+ ?>
+
+
+<!DOCTYPE html>
+<html>
+    <head>
+        <link href="https://fonts.googleapis.com/css?family=Anton&display=swap" rel="stylesheet">
+        <link rel="stylesheet" type="text/css" href="normalize.css">
+        <link rel="stylesheet" type="text/css" href="styles.css">
+        <meta name="viewport" content="width=device-width">
+        <title>Record Collection</title>
+    </head>
+    <body>
+        <header>
+            <h1>Record Collection</h1>
+        </header>
+    <main>
+
+        <?php
+        echo $recordHtml;
+        ?>
+
+    </main>
+    <footer>
+        <a class="addRecord" href="#">ADD A RECORD TO THE COLLECTION</a>
```

Comment:
would want the 'add a record' link in story 1 as then if deploying story 1 it would look incomplete (but i suppose for the sake of this project now leave it in to save time!!) . :)